docs/sinco/v3.x/2_tutorials/8_using_within_docker.md

```diff
@@ -0,0 +1,156 @@
+# Using Within Docker
+
+## Table of Contents
+
+- [Before You Get Started](#before-you-get-started)
+- [Folder Structure End State](#folder-structure-end-state)
+- [Steps](#steps)
+- [Verification](#verification)
+
+## Before You Get Started
+
+Sinco provides support for running your tests inside Docker. It only requires
+just over half a dozen lines inside a dockerfile to install the chrome-driver
+and you should be ready to go.
+
+Other than that, there are no other changes required, and the rest of Sinco's
+documentation still applies.
+
+The ability to run your Browser tests from within a docker container is very
+useful, because (as containers can be networked together), you could go to your
+website, test your /register page works when creating a user, and then clean up
+(delete the user) from your database - whether that's importing your 'User
+model' and deleting it manually.
```

Comment:
```suggestion
The ability to run your Browser tests from within a Docker container is very
useful, because (as containers can be networked together), you could go to your
website, test your `/register` page works when creating a user, and then clean up
(delete the user) from your database -- whether that is importing your "User
model" and deleting it manually.
```